test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1904,3 +1876,22 @@ func checkCpuGovernorsAndResumeLatency(cpus []int, targetNode *corev1.Node, pm_q
 	}
 	return nil
 }
+
+// checkHardwareCapability Checks if test is run on baremetal worker
+func checkHardwareCapability(workerRTNodes []corev1.Node) {
+	for _, node := range workerRTNodes {
+		numaInfo, err := nodes.GetNumaNodes(&node)
+		Expect(err).ToNot(HaveOccurred())
+		if len(numaInfo) < 2 {
+			Skip(fmt.Sprintf("This test need 2 NUMA nodes.The number of NUMA nodes on node %s < 2", node.Name))
+		}
+		// Additional check so that test gets skipped on vm with fake numa
+		onlineCPUCount, err := nodes.ExecCommandOnNode([]string{"nproc", "--all"}, &node)
+		Expect(err).ToNot(HaveOccurred())
+		onlineCPUInt, err := strconv.Atoi(onlineCPUCount)
+		Expect(err).ToNot(HaveOccurred())
+		if onlineCPUInt < 80 {
+			Skip(fmt.Sprintf("This test needs more than 32 CPUs online to work correctly, current online CPUs are %s < 80", onlineCPUCount))
```

Comment:
The string doesn't match the check. Could use a variable or constant so the string always matches the check.

Reply:
addressed in the latest commit.